### PR TITLE
Change description for the operator

### PR DESCRIPTION
**Pull request by piyush-nimbalkar**
- Add more content to the description 
- Change docs URL
- Add UI descriptors for certain for better display in Openshift form view

- Disallow installing operator in all namespaces to avoid confusion. Now install the operator only in single namespace.

- Update the release manifest with new versions.

### CODE REVIEW

File: deploy/olm-catalog/portworx/portworxoperator.v1.1.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -69,71 +69,88 @@ spec:
   - name: Product Features
     url: https://portworx.com/products/features
   - name: Documentation
-    url: https://docs.portworx.com
+    url: https://docs.portworx.com/portworx-install-with-kubernetes/on-premise/openshift/operator
```
Comment (harsh-px) [inline deploy/olm-catalog/portworx/portworxoperator.v1.1.0.clusterserviceversion.yaml]: the OLM framework appears to be generic and not tied to just openshift. Would it be okay to just link to openshift docs in that case?

Comment (piyush-nimbalkar) [reply deploy/olm-catalog/portworx/portworxoperator.v1.1.0.clusterserviceversion.yaml]: The specs from this directory only get published to openshift certified repo. For the generic OLM portworx operator we publish from here -https://github.com/operator-framework/community-operators/tree/master/upstream-community-operators/portworx
with slight modifications.
Will make it a point to not update this link in the community operators repo PR.

File: deploy/olm-catalog/portworx/portworxoperator.v1.1.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -69,71 +69,88 @@ spec:
...
     mediatype: image/png
   description: |
-    ### Storage Cluster Spec Generator
+    Portworx-Enterprise is the most widely-used and reliable cloud-native
```
Comment (adityadani) [inline deploy/olm-catalog/portworx/portworxoperator.v1.1.0.clusterserviceversion.yaml]: Has this been vetted by the marketing team?

File: deploy/olm-catalog/portworx/portworxoperator.v1.1.0.clusterserviceversion.yaml [hunk 3]
```diff
@@ -69,71 +69,88 @@ spec:
...
 
     ### About Portworx
 
-    Portworx Enterprise is the most widely-used and reliable cloud-native
-    storage solution for production workloads and provides high-availability,
-    data protection and security for containerized applications.
+    Portworx is the solution for running stateful containers in production,
+    designed with DevOps in mind. With Portworx, users can manage any database
+    or stateful service on any infrastructure using any container scheduler,
+    including Kubernetes, Mesosphere DC/OS, and Docker Swarm. Portworx solves
```
Comment (adityadani) [inline deploy/olm-catalog/portworx/portworxoperator.v1.1.0.clusterserviceversion.yaml]: Do we need to mention Mesos and Swarm here?

File: deploy/olm-catalog/portworx/portworxoperator.v1.1.0.clusterserviceversion.yaml [hunk 4]
```diff
@@ -69,71 +69,88 @@ spec:
...
+    To uninstall Portworx completely without wiping the data, you should add the
+    following delete strategy to the StorageCluster spec:
     ```
     spec:
       deleteStrategy:
         type: Uninstall
     ```
-    To uninstall Portworx and **wipe all the data**, you should use the following delete strategy:
+    To uninstall Portworx and **wipe all the data**, you should use the following
+    delete strategy:
```
Comment (adityadani) [inline deploy/olm-catalog/portworx/portworxoperator.v1.1.0.clusterserviceversion.yaml]: May be add a warning / caution here.

File: deploy/olm-catalog/portworx/portworxoperator.v1.1.0.clusterserviceversion.yaml (hunk 2 above)
Comment (piyush-nimbalkar) [reply deploy/olm-catalog/portworx/portworxoperator.v1.1.0.clusterserviceversion.yaml]: All the contents in the description is from Vick.

**Pull request merged**